### ISSUE TITLE
Replace CAIA with ADE on research Planning page

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#104445]
### Description
We need to replace references to CAIA on the research Planning page to reflect the new Accessibility Digital Experience (ADE) team.

Please replace `You can request an accessibility review from our CAIA Accessibility team ([#accessibility-help](https://dsva.slack.com/channels/accessibility-help) on Slack)` with `You can request an accessibility review from the Accessibility Digital Experience (ADE) team on Slack using their @accessibility-de handle in [#accessibility-help](https://dsva.slack.com/channels/accessibility-help)`

### Relevant URLs
https://depo-platform-documentation.scrollhelp.site/research-design/planning-moderated-studies

### Which type of team are you on? (Platform team, VFS team, or Leadership)
Platform (ADE)

**Comment by jknipes** [comment]
Hi @coforma-jamie , thanks for your content suggestions. The Platform Content team will review your request and put it in our backlog to refine for next steps.

**Comment by jknipes** [comment]
work completed in [103072](https://github.com/department-of-veterans-affairs/va.gov-team/issues/103072), closing as _Done_.
